lib/utils.js

```diff
@@ -38,27 +38,44 @@ exports.makeGetStationResponseText = function (trainArrivals, stationName) {
     responseText = props.stationNoArrivalsSpeechOutput;
   } else if (_.size(trainArrivals) === 1) {
     var destinationStationName = _.keys(trainArrivals)[0];
-    responseText = 'The next train to ' + destinationStationName + ' arrives in ' +
+    responseText = this.replaceAbbreviations('The next train to ' + destinationStationName + ' arrives in ' +
       trainArrivals[destinationStationName] + ' minutes. There are no other trains arriving at ' + stationName +
-      ' in the next 20 minutes.';
+      ' in the next 20 minutes.', abbreviations['arrivals']);
   } else {
-    responseText = 'Are you going to ' + this.joinListConjuction(_.keys(trainArrivals), ', ', ' or ') + '?';
+    responseText = 'Are you going to ' + this.joinListConjunction(_.keys(trainArrivals), ', ', ' or ') + '?';
   }
   return responseText;
 };
 
-exports.sanitizeServiceAdvisories = function (serviceAdvisoryText) {
-  var replacements = {
-    btwn: 'between',
-    Blu: 'Blue',
-    Org: 'Orange'
-  };
+exports.makeGetDestinationResponseText = function (arrivalTimes, destinationStationName) {
+  var brdOrArr = _.remove(arrivalTimes, function (arrivalTime) {
+    return arrivalTime === 'BRD' || arrivalTime === 'ARR';
+  });
+  var responseText = '';
+  if (_.size(brdOrArr) > 0) {
+    // in the unlikely even that there are back to back BRD and ARR, just tell them about BRD. they won't miss anything
+    responseText += this.replaceAbbreviations('The next train to ' + destinationStationName + ' is ' + brdOrArr[0] +
+      ' now.', abbreviations['arrivals']);
+    if (_.size(arrivalTimes) > 0) {
```

Comment:
Yea the arrival times are only for one station. I don't like how complicated it is either. Once we switch to ES6 and template strings it will be cleaner.